project/AppDependencies.scala

```diff
@@ -3,31 +3,27 @@ import sbt._
 
 object AppDependencies {
 
-  lazy val bootstrapPlayVersion = "7.21.0"
-  lazy val jsoupVersion = "1.13.1"
-  lazy val scalaCheckVersion = "1.14.0"
-  lazy val seleniumVersion = "4.8.3"
+  lazy val bootstrapPlayVersion = "8.4.0"
+  lazy val jsoupVersion         = "1.13.1"
+  lazy val scalaCheckVersion    = "1.14.0"
+  lazy val seleniumVersion      = "4.8.3"
 
   def apply(): Seq[ModuleID] = dependencies ++ testDependencies
 
   lazy val dependencies = Seq(
-    "uk.gov.hmrc"       %%  "bootstrap-frontend-play-28"    % bootstrapPlayVersion,
-    "uk.gov.hmrc"       %%  "play-frontend-hmrc"            % "7.19.0-play-28"
+    "uk.gov.hmrc" %% "bootstrap-frontend-play-30" % bootstrapPlayVersion,
+    "uk.gov.hmrc" %% "play-frontend-hmrc-play-30" % "8.4.0"
```

Comment:
Should this also be `bootstrapPlayVersion`?

Reply:
no, different repo just happens to have the same version.